cmd/vocli/commands/account.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+)
+
+var accInfoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Get information about an account from the vochain. The address may or may not include the 0x prefix",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		c, err := client.New(gatewayRpc)
+		if err != nil {
+			return err
+		}
+		resp, err := c.GetAccount(nil, common.HexToAddress(args[0]))
+		if err != nil {
+			return err
+		}
+		fmt.Fprintln(Stdout, "resp", resp.String(), "err", err)
+		return err
+	},
+}
+
+var accSetInfoCmd = &cobra.Command{
+	Use:   "set <keystore, ipfs://info-URI>",
+	Short: "Create/Update an Account on the vochain",
+	Args:  cobra.ExactArgs(2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		key, _, err := openKeyfile(args[0], "Please unlock your key: ")
+		if err != nil {
+			return err
+		}
+		infoUri := args[1]
+		if !strings.HasPrefix(infoUri, "ipfs://") {
+			return fmt.Errorf("info URI must start with ipfs://, got %s", infoUri)
+		}
+
+		if err := createAccount(key, gatewayRpc, infoUri, faucetHex); err != nil {
+			return err
+		}
+		return nil
+	},
+}
+
+var accCmd = &cobra.Command{
+	Use:   "account",
+	Short: "Manage the Account information stored in the vochain for a particular key",
```

Comment:
If you are going to create an account the account is not yet stored in the Vochain, so I think we should change this command description

Reply:
`Manage a key's Account. Accounts are stored on the vochain.`
How about this?